nextcloud_update.sh

```diff
@@ -547,7 +547,7 @@ then
     mkdir -p "$VMLOGS"
 fi
 
-CURRENTVERSION_after=$(occ_command_no_check status | grep "versionstring" | awk '{print $3}')
+CURRENTVERSION_after=$(occ_command status | grep "versionstring" | awk '{print $3}')
```

Comment:
Why this change?

The update script is run in the startup-script (last step) and needs to work at all times. This wouldn't allow to pass if there was an error.

Reply:
oh, I thought it was this way before. Can change it back.

BTW: wouldn't it make sense to use occ_command here: I imagine that this only fails, if occ isn't responding or existing. That would cause it to fail which it should, shouldn't it?

Reply:
btw: it was this way before...

Reply:
Please look here:
https://github.com/nextcloud/vm/blob/78a458cec7da2cf81d1b0e27290b5217a898142d/nextcloud_update.sh#L524

Reply:
OK, let's keep it as is then.